### DISCUSSION TITLE
Can I perform an unattended install from a bash script?

### DISCUSSION THREAD

**Discussion by m4l490n** [eclipse-cdt/cdt#250] [Q&A]
I'm creating an automation script that needs to get eclipse CDT. I can obviously run the eclipse installer with `./eclipse-inst` but then I have to manually select some options.

I was wondering if I can run the installer with any parameters in order to have it install without any user prompt.

I would prefer to use the installer since it includes the JRE which is much nicer than installing that separately.

Is this even possible?

**Comment by jonahgraham** [comment]
It sounds like just getting the install as a zip/tar may be the easiest? You can download it and just extract it: https://www.eclipse.org/downloads/packages/release/2022-12/r  - Those downloads include a JRE.

**Reply by m4l490n** [reply]
Yeah, but I can't pass any input parameters to the installer to indicate the version I want or the installation folder and I need to be able to do that for the script to be unattended.

**Reply by jonahgraham** [reply]
Ok. The CDT project doesn't handle the installer, we just make the plugins here. Your best bet is looking at the Oomph project. However to the best of my knowledge you can not script it.
